CanvasPlusPlayground/Features/Courses/CourseListView.swift

```diff
@@ -224,13 +239,38 @@ private struct CourseListCell: View {
                     course.rgbColors = .init(color: resolvedCourseColor)
                 }
         }
+        .popover(isPresented: $showRenameTextField) {
+            
+                TextField("New name", text: $renameCourseFieldText)
+                    .onSubmit {
+                        showRenameTextField = false
+                    }
+                    .onDisappear {
+                        course.nickname = renameCourseFieldText
+                        renameCourseFieldText = ""
+                    }
+            
+            .padding(15)
+            
+        }
+        
```

Comment:
hmm idk how it works when you have a binding attached to two presentations (`alert` and `popover`). It think we can just use the `alert`, on both macOS and iOS.

Reply:
This is it as an alert on macOS. Not too bad imo

<img width="1012" alt="Screenshot 2024-12-11 at 4 50 53 PM" src="https://github.com/user-attachments/assets/ec9f07cd-02f9-452d-980e-cf8db374e458" />

Reply:
I like the alert! Looks good.